Userland/Libraries/LibGfx/Vector3.h

```diff
@@ -55,6 +55,11 @@ class Vector3 final {
         return Vector3(m_x - other.m_x, m_y - other.m_y, m_z - other.m_z);
     }
 
+    constexpr Vector3 operator-() const
+    {
+        return Vector4(-m_x, -m_y, -m_z);
+    }
```

Comment:
```suggestion
    constexpr Vector3 operator-() const
    {
        return Vector3(-m_x, -m_y, -m_z);
    }
```
How does this even compile? :thinking:

Reply:
Hmmm, I thought I fixed that one up :thinking: must've been a bad squash

Reply:
Okay this is fixed :sweat_smile: